modules/microshift-custom-ca-proc.adoc

```diff
@@ -0,0 +1,141 @@
+[id="microshift-custom-cas-configuring_{context}"]
+= Configuring custom certificate authorities
+
+To configure externally generated certificates and domain names using custom certificate authorities (CAs), add them to the {microshift-short} `/etc/microshift/config.yaml` configuration file. You must also configure the host operating system trust root.
+
+[NOTE]
+====
+Externally generated `kubeconfig` files are created in the `/var/lib/microshift/resources/kubeadmin/<hostname>/kubeconfig` directory. If you need to use `localhost` in addition to externally generated configurations, retain the original `kubeconfig` file in its default location. The `localhost` `kubeconfig` file uses the self-signed certificate authority.
+====
+
+.Prerequisites
+* The OpenShift CLI (`oc`) is installed.
+* You have access to the cluster as a user with the cluster administration role.
+* The certificate authority has issued the custom certificates.
+* A {microshift-short} `/etc/microshift/config.yaml` configuration file exists.
+
+.Procedure
+
+. Copy the custom certificates you want to add to the trust root of the {microshift-short} host. Ensure that the
+certificate and private keys are only accessible to {microshift-short}.
+
+. For each custom CA that you need, add an `apiServer` section called `namedCertificates` to the `/etc/microshift/config.yaml` {microshift-short} configuration file by using the following example:
++
+[source,yaml]
+----
+apiServer:
+  namedCertificates:
+   - certPath: ~/certs/api_fqdn_1.crt <1>
+     keyPath:  ~/certs/api_fqdn_1.key <2>
+   - certPath: ~/certs/api_fqdn_2.crt
+     keyPath:  ~/certs/api_fqdn_2.key
+     names: <3>
+     - api_fqdn_1
+     - *.apps.external.com
+----
+<1> Add the full path to the certificate.
+<2> Add the full path to the certificate key.
+<3> Optional. Add a list of explicit DNS names. Leading wildcards are allowed. If no names are provided, the implicit names are extracted from the certificates.
+
+. Restart the {microshift-service} to apply the certificates by running the following command:
++
+[source,terminal]
+----
+$ systemctl microshift restart
+----
+
+. Wait a few minutes for the system to restart and apply the custom server. New `kubeconfig` files are generated in the `/var/lib/microshift/resources/kubeadmin/` directory.
+
+. Copy the `kubeconfig` files to the client. If you specified wildcards for the IP address, update the `kubeconfig` to remove the public IP address of the server and replace that IP address with the specific wildcard range you want to use.
+
+. From the client, use the following steps:
+
+.. Specify the `kubeconfig` to use by running the following command:
++
+[source,terminal]
+----
+$ export KUBECONFIG=~/custom-kubeconfigs/kubeconfig <1>
+----
+<1> Use the location of the copied `kubeconfig` file as the path.
+
+.. Check that the certificates are applied by using the following command:
++
+[source,terminal]
+----
+$ oc --certificate-authority ~/certs/ca.ca get node
```

Comment:
@ShaunaDiaz for me this is not working
oc --certificate-authority /tmp/caCert.pem get node

Reply:
Sorry it works fine I misplaced new kubeconfig